hack/project_info.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -e
+
+#
+# Copyright 2019 IBM Corp.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+operator_yaml_path=deploy/installer/generated/ibm-block-csi-operator.yaml
+
+get_current_csi_version (){
+  current_csi_version=$(cat version/version.go | grep -i driverversion | awk -F = '{print $2}')
+  echo ${current_csi_version//\"}
+}
+
+are_manifest_files_exsists_in_current_csi_version (){
+  current_csi_version=$(get_current_csi_version)
+  if ! compgen -G "${PWD}/deploy/olm-catalog/*/$current_csi_version" > /dev/null; then
+    exit 0
+  fi
+}
+
+get_csv_files (){
+  current_csi_version=$(get_current_csi_version)
+  ls deploy/olm-catalog/*/$current_csi_version/manifests/ibm-block-csi-operator.v$current_csi_version.clusterserviceversion.yaml
```

Comment:
please [avoid wildcards](https://github.com/IBM/ibm-block-csi-operator/pull/220#discussion_r744253629)

Reply:
I [still see the wildcard](https://github.com/IBM/ibm-block-csi-operator/blob/5f6371694ab49490297168fe2b3765156072b9dc/hack/project_info.sh#L29).
what happens if there's only one bundle for the current csi version? I don't think the test should fail. it should test what it can, and allow separate additions